src/views/Pools/components/CakeVaultCard/index.tsx

```diff
@@ -96,15 +96,14 @@ const CakeVaultCard: React.FC<CakeVaultProps> = ({ pool, showStakedOnly, default
   const { t } = useTranslation()
   const { account } = useWeb3React()
   const {
-    userData: { userShares, isLoading: isVaultUserDataLoading },
+    userData: { userShares, isLoading },
     fees: { performanceFeeAsDecimal },
     pricePerFullShare,
   } = useVaultPoolByKey(pool.vaultKey)
 
   const { cakeAsBigNumber } = convertSharesToCake(userShares, pricePerFullShare)
 
   const accountHasSharesStaked = userShares && userShares.gt(0)
-  const isLoading = !pool.userData || isVaultUserDataLoading
```

Comment:
This is probably a leftover when there was only one card for cakevaultcard and poolcard